### DISCUSSION TITLE
Support for custom css

### DISCUSSION THREAD

**Discussion by tionis** [alangrainger/share-note#115] [Vote for upcoming features]
It would be nice if custom css from the active css snippets would also be loaded

**Comment by ksdavidc** [comment]
agreed

**Comment by alangrainger** [comment]
Hi both - custom CSS has always been supported.

Here's a custom CSS snippet turning paragraphs to red:

```css
p {
    color: red;
}
```

And here's the resulting note, showing red paragraphs:

https://share.note.sx/z4ik6olx#5Q7/bFHYdrMTR7+eg9xjzgJYJy6JA/OLw1+73zYI7SE

Please make sure to [follow the troubleshooting steps before creating an issue](https://docs.note.sx/troubleshooting). You probably just need to Force re-upload your CSS after adding the custom snippets.
